roles/manage_vpc_peering/README.md

```diff
@@ -0,0 +1,105 @@
+# manage_vpc_peering
+
+A role to create and delete VPC peering connections.
+
+currently works for VPCs under same account and same region.
+
+## Specify following values in role vars
+
+- region - Region of the requester VPC (region in which VPC peering connection request resides when performing a delete operation)
+
+- requester_vpc - ID of the VPC requesting the peering connection.
+
+- accepter_vpc - ID of the VPC accepting the perring connection.
+
+- accepter_vpc_region - Region of the accepter VPC (Required if requester and accepter VPCs are in different regions and performing cross-region peering.)
+
+- accepter_vpc_account_id - The AWS account number of accepter VPC account for cross account peering.
+
+- accepter_account_profile - A Named AWS profile of accepter VPC account for cross account peering.
+
+- vpc_peering_operation - Choices include 'create', 'delete', and 'accept'.
+
+- vpc_peering_conn_id - ID of the VPC peering connection request (only provide to delete a VPC peering connection).
+
+Dependencies
+------------
+
+- role: [aws_setup_credentials](../aws_setup_credentials/README.md)
+
+## Example:
+```
+---
+- name: Playbook for managing VPC peering connections using cloud.aws_ops.manage_vpc_peering role
+  hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: Peer VPCs in same account and region (local)
+      ansible.builtin.include_role:
+        name: cloud.aws_ops.manage_vpc_peering
+      vars:
+        region: us-west-1
+        requester_vpc: vpc-12345
+        accepter_vpc: vpc-98765
+        vpc_peering_operation: create
+
+    - name: Peer VPCs in same account and different region (local cross-region)
+      ansible.builtin.include_role:
+        name: cloud.aws_ops.manage_vpc_peering
+      vars:
+        region: us-west-1
+        requester_vpc: vpc-12345
+        accepter_vpc: vpc-98765
+        accepter_vpc_region: ap-northeast-3
+        vpc_peering_operation: create
+
+    - name: Peer VPCs in same account and different region (cross-account)
```

Comment:
Wouldn't "cross-account" be across different accounts?  Should this take name be cross-region instead?